modeldb/base/service-webapp.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     app: modeldb
     component: webapp
-    service.onepanel.io/name: modeldb
+    service.onepanel.io/name: ModelDB
```

Comment:
is this supposed to be here?

Reply:
@rushtehrani I can make a separate PR for it, but you made a request earlier to change the name to that case.